### ISSUE TITLE
Move and Scale tools are unusable

### ISSUE THREAD

**Issue by Boalbads** [andreja6/G3D-Fun#69]
The tools are unusable, you are unable to select the arrows/balls to move/scale the object.

**Comment by Vulpovile** [comment]
Yeah, they aren't implemented yet. I might keep this issue open or close it if I manage to make the Trello visible by the public
